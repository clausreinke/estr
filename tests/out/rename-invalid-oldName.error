Error: Command failed: not a valid variable name >old-name<
