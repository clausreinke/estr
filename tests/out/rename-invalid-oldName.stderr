not a valid variable name >old-name<
